TTHAnalysis/python/plotter/susy-ewkino/2lss/systs_ewkino_2lss.txt

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Why a separate systs file? The idea was to use the common one in susy-ewkino/systs_ewkino.txt